src/components/DappBrowser/BrowserTab.tsx

```diff
@@ -810,7 +810,7 @@ export const BrowserTab = React.memo(function BrowserTab({ tabId, tabIndex, inje
                   )}
                 </View>
               </ViewShot>
-              <AnimatedFasterImage source={screenshotSource.value} style={[styles.screenshotContainerStyle, animatedScreenshotStyle]} />
+              <AnimatedFasterImage source={screenshotSource} style={[styles.screenshotContainerStyle, animatedScreenshotStyle]} />
```

Comment:
pretty sure @benisgold  made this change a day ago

Reply:
I had seen that change in Ben's PR the other day, tested it because I suspected it might break the instant setting, then accidentally included it in my tab-view-transitions branch. Synced with Ben on it earlier today.